website/src/components/Changelog/Apple.tsx

```diff
@@ -24,6 +24,9 @@ export default function Apple() {
           Fixes a rare crash that could occur when dismissing the update
           available notification.
         </ChangeItem>
+        <ChangeItem pull="8126">
+          Mitigates a crash in the error handling code of the Network Extension.
```

Comment:
We may not even need this. If this happens, the session is disconnecting anyway, and the user probably doesn't know it's crashed. The GUI app will have survived just fine. WDYT?